src/Config.php

```diff
@@ -330,73 +328,77 @@ private static function init()
         ]);
 
         if (!self::atLeast(8, 7)) {
-            throw new Vips\Exception("your libvips is too old -- " .
+            throw new Exception("your libvips is too old -- " .
                 "8.7 or later required");
         }
 
-        if (PHP_INT_SIZE != 8) {
-            # we could maybe fix this if it's important ... it's mostly
-            # necessary since GType is the size of a pointer, and there's no
-            # easy way to discover if php is running on a 32 or 64-bit
-            # systems (as far as I can see)
-            throw new Vips\Exception("your php only supports 32-bit ints -- " .
-                "64 bit ints required");
-        }
+        $is_64bits = PHP_INT_SIZE === 8;
```

Comment:
Is this always true? You can configure PHP to use 64-bit ints on a 32-bit machine, I believe. Maybe it's true enough to not matter!

Reply:
It looks like this is being determined at compile-time of PHP based on the current system. See:
https://github.com/php/php-src/blob/eb5b12c2255238f99adf1be06c46b7a1276311e4/main/main.c#L2139
https://github.com/php/php-src/blob/eb5b12c2255238f99adf1be06c46b7a1276311e4/Zend/zend_long.h#L50
https://github.com/php/php-src/blob/eb5b12c2255238f99adf1be06c46b7a1276311e4/Zend/zend_long.h#L25-L28

It could theoretically be overwritten by doing something like `-D__x86_64__=1`, but I'm not aware of any systems that mimic 64-bit support when it's actually 32-bit.

Reply:
Ah OK, yes that looks pretty safe.